index.html

```diff
@@ -63,6 +63,9 @@
         functions such as qualifying new hardware. It's an open-source project
         under the GPL and is used and developed by a number of organizations,
         including Google, IBM, Red Hat, and many others.</p>
+        <p style="color:red;">Autotest in being discontinued in favor of the next
+        generation testing framework,
+        <a href="http://avocado-framework.github.io/">Avocado Framework</a>.</p>
```

Comment:
Yep, that looks like what I imagined. I'm not native, but I think the English should be improved a bit. (Not to say strictly discontinued, but more-like the maintaners moved to newer project avocado, which has the same purpose, only it's suppose to be cleaner and eventually more powerful).